### PR TITLE
Rename EventTypes: Task -> Rally, delete EventManaged

### PR DESCRIPTION
**Pull request by robrich**
TODO: what to do with EventManaged-specific logic?

### CODE REVIEW

**Comment by robrich** [comment]
FRAGILE: Don't merge this yet.  I'd like to discuss how we handle the commented-out EventManaged logic before we push this into place.

**Comment by tonysurma** [comment]
For now, delete and we can always bring it back in.

Also i if you prefix the pr name with wip while it's a work in progress that's our convention to know to not merge yet

File: AllReadyApp/Web-App/AllReady/Models/EventType.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace AllReady.Models
+{
+    public enum EventType
```
Comment (MisterJames) [inline AllReadyApp/Web-App/AllReady/Models/EventType.cs]: yay for non-pluralized enums! :)

File: AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Tasks/AssignTaskCommandHandlerAsync.cs [hunk 2]
```diff
@@ -69,7 +69,7 @@ public AssignTaskCommandHandlerAsync(AllReadyContext context, IMediator mediator
                 var taskSignupsToRemove = task.AssignedVolunteers.Where(taskSignup => message.UserIds.All(uid => uid != taskSignup.User.Id)).ToList();
                 taskSignupsToRemove.ForEach(taskSignup => task.AssignedVolunteers.Remove(taskSignup));
 
-                if (campaignEvent.EventType != EventTypes.EventManaged)
+                if (true /*campaignEvent.EventType != EventTypes.EventManaged*/)
```
Comment (MisterJames) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Tasks/AssignTaskCommandHandlerAsync.cs]: If deleting the EventManaged per @tonysurma I'm sure we can remove the conditional blocks as appropriate (which I think was related to your query on if we should keep them). Should be safe to remove.

**Comment by MisterJames** [comment]
@robrich I am good with direction here and agree with @tonysurma on the delete note. If you clean up those conditional bits we should be good to go. Also +1 on the depluralization ;)

**Comment by robrich** [comment]
Should be good to go now.  Merge when ready.

**Comment by tonysurma** [comment]
:shipit:

**Comment by robrich** [comment]
How are you rebasing these as you merge?  I've not found such a button on GitHub's site.  Awesome, btw.

**Comment by robrich** [comment]
Will this cause stale data issues for any events currently stored in the db as `EventType=1`?

**Comment by tonysurma** [comment]
@robrich if you can verify no sample data is injected (I doubt it as you would have found it already) I will verify database (on my list of things to do)

**Comment by robrich** [comment]
I patched sample data generation as I made the changes as they're compile time errors.  :D

**Comment by tonysurma** [comment]
yep I figured as such

**Pull request merged**